README.md

```diff
@@ -1,19 +1,11 @@
 EasySplash
 ==========
 
-This is a simple program for animated splash screens using OpenGL ES for rendering. It consists
-of the main splash screen program (easysplash) and a small control program (easysplashctl).
+This is a simple program for animated splash screens using GStreamer for rendering. It consists of
+an application which is capable of playing the animation and control its flow.
 
-EasySplash takes as input zip archives containing a description and PNG-encoded image frames.
-The frames are divided in parts. The archive format is documented [here](doc/Animation-Structure.md)
-
-It is possible to run EasySplash in X11 and on the PC. This is useful for debugging purposes
-and for trying out new animations on the PC before uploading it to embedded devices.
-
-EasySplash can run in realtime or non-realtime mode. realtime mode means the animaton progresses
-in realtime, without user interaction. In non-realtime mode, the animation progresses only by
-sending progress updates over easysplashctl. The latter is useful if a progress value can be
-determined while something is loading, for example.
+It is possible to run EasySplash in X11 and on the PC. This is useful for debugging purposes and for
```

Comment:
"possible to run [..] in X11 and on the PC" seems a bit redundant here.

Reply:
Done.